Python/bytecodes.c

```diff
@@ -3345,25 +3345,29 @@ dummy_func(
             _FOR_ITER_GEN_FRAME +
             _PUSH_FRAME;
 
-        inst(LOAD_SPECIAL, (owner -- attr, self_or_null)) {
-            assert(oparg <= SPECIAL_MAX);
-            PyObject *owner_o = PyStackRef_AsPyObjectSteal(owner);
+        op(_INSERT_NULL, (arg -- args[2])) {
```

Comment:
@markshannon - this is what I was referring to in our last meeting. I had to split up `LOAD_SPECIAL` into two ops to ensure that that `frame->stackpointer` includes both operands when it's spilled before the `_PyObject_LookupSpecialMethod` call.

I'm not sure if this is a better way to do this.